endpoint/test_endpoint.c

```diff
@@ -12,7 +12,7 @@
 #include <time.h>
```

Comment:
Rename the file name to test_endpoint_core

Reply:
Done.